ci/windows/win_build_test.ps1

```diff
@@ -69,6 +69,7 @@ if (!$llvmVer.CompareTo("10")) {
     $env:TAICHI_CMAKE_ARGS += " -DCLANG_EXECUTABLE=C:\\taichi_clang_15\\bin\\clang++.exe"
 	$env:TAICHI_CMAKE_ARGS += " -DLLVM_AS_EXECUTABLE=C:\\taichi_llvm_15\\bin\\llvm-as.exe"
 	$env:TAICHI_CMAKE_ARGS += " -DTI_LLVM_15:BOOL=ON"
+    $env:TAICHI_CMAKE_ARGS += " -DTI_WITH_DX12:BOOL=ON"
```

Comment:
This file is obsolete so you don't have to add this, it will be removed.

Reply:
Which file should I use to enable DX12?

Reply:
In `testing.yml`, add to `TAICHI_CMAKE_ARGS` environment variable to `build_and_test_windows` task

Reply:
![2022-09-26-144139_766x845_scrot](https://user-images.githubusercontent.com/857880/192209936-8046902b-878e-42ab-b26d-2fa2d4c5be2d.png)

Reply:
![2022-09-26-144250_753x889_scrot](https://user-images.githubusercontent.com/857880/192210089-062e0d1f-fda4-4c88-ba88-9f082a57d909.png)